neuralmonkey/encoders/attentive.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=too-few-public-methods
+class Attentive(object):
+    def __init__(self, attention_type, **kwargs):
+        self._attention_type = attention_type
+        self._attention_kwargs = kwargs
+
+        assert hasattr(self, "name")
+        assert hasattr(self, "_padding")
+        assert hasattr(self, "_attention_tensor")
+
+    def get_attention_object(self, runtime: bool=False):
+        # pylint: disable=no-member
+        if self._attention_type and self._attention_tensor is None:
+            raise Exception("Can't get attention: missing attention tensor.")
```

Comment:
Dal bych `ValueError` páč to má blbou hodnotu. Nebo vytvořit v tomhle modulu nějakej specifičtější error.